src/controllers/listItemController.js

```diff
@@ -0,0 +1,49 @@
+const dataService = require('../services/dataService/listItemFunctions')
+
+function findListItem(req, res, next) {
+  dataService.findListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function createListItem(req, res, next) {
+  dataService.createListItem()
+    .then(listsItem => {
+      res.status(200).json(listsItem)
+    })
+    .catch(next)
+}
+
+function deleteListItem(req, res, next) {
+  dataService.deleteListItem()
```

Comment:
Which list item? The client would have been expected to specify a list item ID in the url `/items/:itemId`. We should be grabbing this ID and passing it into `dataService.deleteListItem(listItemId)`.